docs/src/main/asciidoc/getting-started-testing.adoc

```diff
@@ -504,11 +505,19 @@ public class MockGreetingProfile implements QuarkusTestProfile {
     public String getConfigProfile() { <3>
         return "test";
     }
+
+    @Override
+    public List<TestResourceEntry> testResources() { <4>
```

Comment:
I think you're missing an import to `java.util.List`?

Reply:
Fixed